src/api/libcellml/variable.h

```diff
@@ -39,6 +39,20 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     Variable& operator=(Variable n); /**< Assignment operator */
 
     /**
+     * @brief The INTERFACE_TYPES enum.
+     * Enum describing the interface types that are available for
+     * a given Variable.
+     */
+    enum INTERFACE_TYPES
+    {
+        INTERFACE_TYPE_UNDEFINED,
```

Comment:
The current use cases can be met with semantically equivalent serialisation, so I think it is fine for now to have `none` and drop the `undefined`. Some of the latter parts of the roadmap currently require maintaining a particular serialisation, so we might need to revisit this later.